config/initializers/devise.rb

```diff
@@ -9,7 +9,7 @@
   config.skip_session_storage = [:http_auth]
   config.stretches = Rails.env.test? ? 1 : 10
   config.reconfirmable = true
-  config.password_length = 4..128
+  config.password_length = 8..128
```

Comment:
@parrish as discussed I don't think this will interfere with any legacy / ouroboros login but just when updating the user model via panoptes API if the password field is present (change), [see here](https://github.com/plataformatec/devise/blob/6720f93d2c54904f10b6e6d6beb51d0b5b330976/lib/devise/models/validatable.rb#L35)